### PR TITLE
adds GitHub Actions and build containers

### PR DESCRIPTION
**Pull request by m00sey**
This PR supersedes #1622 

Signed-off-by: Kevin Griffin <griffin.kev@gmail.com>

### CODE REVIEW

**Comment by WadeBarnes** [comment]
(ci) test this please

**Comment by WadeBarnes** [comment]
(ci) test this please

**Review by WadeBarnes — APPROVED** [review]

File: ci/ubuntu.dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ARG venv=venv
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
     libindy-crypto=0.4.5 \
-    libindy=1.13.0~1420 \
+    libindy=1.15.0 \
```
Comment (WadeBarnes) [inline ci/ubuntu.dockerfile:11]: Looks like this is causing a Jenkins build failure.
```
Version '1.15.0' for 'libindy' was not found
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

**Comment by WadeBarnes** [comment]
(ci) test this please

**Review by WadeBarnes — APPROVED** [review]

**Review by Toktar — APPROVED** [review]

**Pull request merged**